packages/server/graphql/mutations/createTaskIntegration.ts

```diff
@@ -134,11 +124,33 @@ export default {
       return {error: {message: createTaskResponse.message}}
     }
 
+    const {issueId, ...updateTaskInput} = createTaskResponse
+
+    if (
+      userId &&
+      viewerId !== userId &&
+      'addCreatedBySomeoneElseComment' in taskIntegrationManager
+    ) {
+      const addCommentResponse = await taskIntegrationManager.addCreatedBySomeoneElseComment(
```

Comment:
+1 interesting! so instead of tagging this to the bottom of the task we're adding it as a comment?

Reply:
@mattkrick Yes, we already did that for GitHub and Jira. Probably, because adding it to the text will show it in the card on a Parabol board as well?